templates/aws-stack.yml

```diff
@@ -266,6 +272,11 @@ Parameters:
     Description: Optional - Comma separated list of managed IAM policy ARNs to attach to the instance role
     Default: ""
 
+  LambdaManagedPolicyARN:
```

Comment:
I'd prefer to not add a new configuration parameter for this, is it needed?

Reply:
It could be left to something external to this CloudFormation but I've added it here mainly for if the SSM parameter is encrypted, the lambda will need `kms:Decrypt` permission on the key. Other option I see is to export details about the lambda so something else can get the role ARN and attach a policy to it. The issue is that means something else will be changing resources created from this stack which probably isn't ideal

Reply:
Can we just add a policy with those permissions to the template?

Reply:
Certainly. The issue is that I think we'll need to add another parameter just for the KMS key ARN, since I envisage this policy to grant `ssm:GetParamter` and `kms:Decrypt`. We will be able to construct the ARN for the parameter, but not the KMS key. Are you okay with adding another parameter in that case?

Reply:
I’m ok with a parameter specifically for the KMS Key ID for the Parameter Store Path.